src/cmd_line.c

```diff
@@ -49,6 +51,11 @@ void cmd_line_attempt(const char *line) {
         cmd_line_ls(the_rest);
     } else if (strcmp("cat", first_word) == 0) {
         cmd_line_cat(the_rest);
+    } else if (strcmp("process", first_word) == 0) { // temporary, for debugging
+        struct process_permissions test_permissions;
+        test_permissions.max_number_of_pages = 10;
```

Comment:
I'm okay with this, since the permission struct is always copied into the process struct